### PR TITLE
Update whats-new-in-office-365-atp.md

### PR DESCRIPTION
**Pull request by vippand**
Updating what's new to include Top Targeted Users, connectors, and ETRs

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/whats-new-in-office-365-atp.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ This article lists new features in the latest release of Office 365 ATP. Feature
 
 - [Check your policies using Configuration Analyzer](configuration-analyzer-for-security-policies.md)
 - [Manually override filter verdicts using the tenant allow/block list](tenant-allow-block-list.md)
+- [Extended capabilities in Threat Explorer including Top Targeted users, Transport Rules and Connectors](threat-explorer.md#extended-capabilities-in-Threat-Explorer.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-office-365-atp.md]:
```suggestion
- [Extended capabilities in Threat Explorer including top targeted users, transport rules, and connectors](threat-explorer.md#extended-capabilities-in-Threat-Explorer.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — COMMENTED** [review]
Thank you @vippand and @JohanFreelancer9

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Pull request merged**